rlp-derive/src/de.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright 2015-2020 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
```

Comment:
`s/Parity Ethereum/Parity Common` or `s/Parity Ethereum/Parity` as in other files?

Reply:
```
// Copyright 2020 Parity Technologies
//
// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
// option. This file may not be copied, modified, or distributed
// except according to those terms.
```